manpages/0.7.x/man8/zfs.8.html

```diff
@@ -0,0 +1,10752 @@
+<HTML><HEAD><TITLE>Manpage of ZFS</TITLE>
+</HEAD><BODY>
+<H1>ZFS</H1>
+Section: Maintenance Commands (8)<BR>Updated: SMM<BR>
```

Comment:
I noticed a few pages have "Updated: SMM " rather than the correct date.  This looks like a translation issue since there are real dates in the man pages.